website/forms.py

```diff
@@ -53,38 +69,51 @@ def __init__(self, *args, **kwargs):
         category = kwargs.pop('category', None)
         selecttutorial = kwargs.pop('tutorial', None)
 
-        print "category", category
+        # print "category", category
```

Comment:
Print statements are unnecessary. Commented prints, more so! Please remove this.